features/Addtocart.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Add to cart
+
+Background:Add to cart
+	Given the user in Homepage and select on the Product
+    And The user can be navigated to the product details page 
+
+
+    Scenario:Validate Empty Add to cart(Neagtive scenario)
+    When The user clicks on Add to Cart button
+    Then the user should see 2 error messages "This is a required field"
+
+
+
+    Scenario:Out of stock Add to cart(Neagtive scenario)
+    When the user gives more than the stock and clicks Add to cart
```

Comment:
@karthikeyan7010 It can be updated as "**When the user add more stock to cart than the available stock**"